go-backend/db/organization_test.go

```diff
@@ -0,0 +1,91 @@
+package db
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"context"
+	"joshsoftware/peerly/config"
+	"testing"
+	logger "github.com/sirupsen/logrus"
+)
+
+// Define the suite, and absorb the built-in basic suite
+// functionality from testify - including assertion methods.
+type OrganizationTestSuite struct {
+	suite.Suite
+	dbStore Storer
+}
+
+func TestExampleTestSuite(t *testing.T) {
+	suite.Run(t, new(OrganizationTestSuite))
+}
+
+func (suite *OrganizationTestSuite) SetupSuite() {
+	config.Load("application_test")
+
+	err := RunMigrations()
+	if err!=nil {
+		logger.WithField("err", err.Error()).Error("Database init failed")
```

Comment:
return when error

Reply:
or may be fail the test at this point. We dont want to continue to running test cases